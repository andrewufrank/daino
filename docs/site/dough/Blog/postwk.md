---
title: postwk with image
author: AUF
date: Jan. 4, 2019
keywords: test
abstract: A silly text not needing an abstract.
pageTemplate: page3.yaml
---

Some text without meaning; really? with an image 

![first picture ]( /home/frank/Workspace8/ssg/docs/site/dough/resources/img/120-2026_IMG.JPG  "Logo Title Text 1")

absolute statt relative path "/resources/img/120-2026_IMG.JPG"
  An example post.  postwk but in blog - not in SubBlog
with some additional text 
which is always changed  

addition
