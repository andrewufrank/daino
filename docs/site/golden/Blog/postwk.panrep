Panrep {panyam = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/postwk.md", dyLink = "Blog/postwk.md", dyLang = DLenglish, dyTitle = "postwk with image", dyAbstract = "A silly text not needing an abstract.", dyAuthor = "AUF", dyDate = Just "Jan. 4, 2019", dyKeywords = "test", dyBibliography = Nothing, dyStyle = Nothing, dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = False, dyIndexSort = Nothing, dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/postwk.md", link = "Blog/postwk.md", title = "postwk with image", abstract = "A silly text not needing an abstract.", author = "AUF", date = "Jan. 4, 2019", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}}, panpan = Pandoc (Meta {unMeta = fromList [("abstract",MetaInlines [Str "A",Space,Str "silly",Space,Str "text",Space,Str "not",Space,Str "needing",Space,Str "an",Space,Str "abstract."]),("author",MetaInlines [Str "AUF"]),("date",MetaInlines [Str "Jan.",Space,Str "4,",Space,Str "2019"]),("keywords",MetaInlines [Str "test"]),("pageTemplate",MetaInlines [Str "page3.yaml"]),("title",MetaInlines [Str "postwk",Space,Str "with",Space,Str "image"])]}) [Para [Str "Some",Space,Str "text",Space,Str "without",Space,Str "meaning;",Space,Str "really?",Space,Str "with",Space,Str "an",Space,Str "image"],Para [Image ("",[],[]) [Str "first",Space,Str "picture"] ("/home/frank/Workspace11/ssg/docs/site/dough/resources/img/120-2026_IMG.JPG","Logo Title Text 1")],Para [RawInline (Format "html") "<!-- ![first picture ]( ../resources/img/120-2026_IMG.JPG  \"Logo Title Text 1\")  -->"],Para [Str "absolute",Space,Str "statt",Space,Str "relative",Space,Str "path",Space,Str "\"/resources/img/120-2026_IMG.JPG\"",SoftBreak,Str "An",Space,Str "example",Space,Str "post.",Space,Str "postwk",Space,Str "but",Space,Str "in",Space,Str "blog",Space,Str "-",Space,Str "not",Space,Str "in",Space,Str "SubBlog",SoftBreak,Str "with",Space,Str "some",Space,Str "additional",Space,Str "text",SoftBreak,Str "which",Space,Str "is",Space,Str "always",Space,Str "changed"],Para [Str "addition"]]}