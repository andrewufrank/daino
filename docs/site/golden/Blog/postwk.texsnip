TexSnip {snipyam = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/postwk.md", dyLink = "Blog/postwk.md", dyLang = DLenglish, dyTitle = "postwk with image", dyAbstract = "A silly text not needing an abstract.", dyAuthor = "AUF", dyDate = Just "Jan. 4, 2019", dyKeywords = "test", dyBibliography = Nothing, dyStyle = Nothing, dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = False, dyIndexSort = Nothing, dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/postwk.md", link = "Blog/postwk.md", title = "postwk with image", abstract = "A silly text not needing an abstract.", author = "AUF", date = "Jan. 4, 2019", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}}, unTexSnip = "Some text without meaning; really? with an image\n\n\\includegraphics{/home/frank/Workspace11/ssg/docs/site/dough/resources/img/120-2026_IMG.JPG}\n\nabsolute statt relative path \"/resources/img/120-2026\\_IMG.JPG\" An\nexample post. postwk but in blog - not in SubBlog with some additional\ntext which is always changed\n\naddition"}