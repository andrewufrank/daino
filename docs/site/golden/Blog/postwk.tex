\documentclass[a4paper,10pt]{scrbook}
\usepackage{fontspec}
\usepackage[ngerman]{babel}
\usepackage{graphicx}
\usepackage{makeidx}
\usepackage{natbib}
\makeindex
\usepackage[colorlinks]{hyperref}
\providecommand{\tightlist}{%
\setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

\begin{document}

Some text without meaning; really? with an image

\includegraphics{/home/frank/Workspace11/ssg/docs/site/dough/resources/img/120-2026_IMG.JPG}

absolute statt relative path "/resources/img/120-2026\_IMG.JPG" An
example post. postwk but in blog - not in SubBlog with some additional
text which is always changed

addition


\printindex
\end{document}
